h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java

```diff
@@ -483,7 +484,11 @@ protected static JsonElement findInJson(final JsonElement jsonElement, final Str
             } else break;
         }
 
-        return result;
```

Comment:
looks like https://github.com/h2oai/h2o-3/pull/5775/files#diff-4a16d86a307592634a57c770ec83647ad7867d4d373ea304df64f5defc3074cfR475 became redundant

Reply:
yes, you are right. Thank you!